tests/e2e/configs/one-chain.yaml

```diff
@@ -2,6 +2,8 @@ chains:
   - name: osmosis-1
     type: osmosis
     numValidators: 1
+    storageClassName: standard-rwo
+    storage: 1Gi
```

Comment:
could we update this to
```yaml
storage:
  className: standard-rwo
  size: 1Gi
```

Might make more sense for the users. I would like to avoid exposing all the k8s specific naming directly to the users (if it can be avoided and made simpler).

Reply:
I imagine that this is mostly needed for production use case. At least, that's when we use it. With this assumption, I believe that a Starship user knows what they are doing and familiar with k8s concepts and hence they want to have better controls. Otherwise, Starship needs to expose the same concepts but with Starship alternatives. That doesn't sounds like a good UX to me. 

If you have better ideas at how to allow fine controls, let me know.

Reply:
Makes sense, we dont have to change the names, but if we could still move it into storage, it will make the happy path to be less cluttered.